gui/packages/ubuntupro/lib/pages/widgets/navigation_row.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class NavigationRow extends StatelessWidget {
+  const NavigationRow({
+    required this.onBack,
+    required this.onNext,
+    this.backText,
+    this.nextText,
+    super.key,
+  });
+
+  final void Function()? onBack;
+  final String? backText;
+  final void Function()? onNext;
+  final String? nextText;
+
+  @override
+  Widget build(BuildContext context) {
+    final lang = AppLocalizations.of(context);
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        OutlinedButton(
+          onPressed: onBack,
+          child: Text(backText ?? lang.buttonBack),
+        ),
```

Comment:
Also, should the back button be conditionally added to the row? Consider the first page of the wizard: you either avoid using the navigation row and place the button at the same precise location on the page or you use the navigation row but then the back button should not be shown.

Reply:
BTW I'm ok if you plan to tackle this only by the time this becomes necessary, as long as you've got a plan :)